tests/app/conftest.py

```diff
@@ -180,3 +181,31 @@ def _create():
 
     mock_class = mocker.patch('app.dao.users_dao.create_secret_code', side_effect=_create)
     return mock_class
+
+
+@pytest.fixture(scope='function')
+def sample_notification(notify_db,
+                        notify_db_session,
+                        service=None,
+                        template=None,
+                        job=None):
+    if service is None:
+        service = sample_service(notify_db, notify_db_session)
+    if template is None:
+        template = sample_template(notify_db, notify_db_session, service=service)
+    if job is None:
+        job = sample_job(notify_db, notify_db_session, service=service, template=template)
+
+    notificaton_id = uuid.uuid4()
+    to = '+44709123456'
+
+    data = {
+        'id': notificaton_id,
+        'to': to,
+        'job_id': job.id,
+        'service_id': service.id,
+        'template_id': template.id
+    }
```

Comment:
To keep consistent with the other parts of the marshmallow schemas, we should remove the '_id' from job, server and template.